eva-accession-ws/src/test/java/uk/ac/ebi/eva/accession/ws/ClusteredVariantsRestControllerTest.java

```diff
@@ -111,10 +116,15 @@
     private static final int VERSION_1 = 1;
 
     private static final int VERSION_2 = 1;
+    public static final long DBSNP_CLUSTERED_VARIANT_ACCESSION_HUMAN_1 = 1118L;
+    public static final long DBSNP_CLUSTERED_VARIANT_ACCESSION_HUMAN_2 = 1475292486L;
```

Comment:
Any reason just these two variables have to be public?

Reply:
The auto-generation of constants by default create them as public and forgot to modify it. Changed